tests/__init.test.js

```diff
@@ -5,6 +5,24 @@
  */
 window.debug = true;
 
+// getVRDisplays mock
+navigator.getVRDisplays = function () {
```

Comment:
This stil does not make it. You are no attaching requestPreset/exitPresent to a mock vrDisplay that the promise will return.